### ISSUE TITLE
Memory leak?

### ISSUE THREAD

**Issue by hanseul1795** [kelseyhightower/nocode#1859]
`       `

please fix.

**Comment by cybrox** [comment]
According to the docs, `   ` is deprecated and has been replaced by `   `.
This should definitely be fixed, since apparently `   ` can lead to memory leaks when still used.

**Comment by dogebit** [comment]
I totally agree with @cybrox, except sometimes you can even get away by doing ` `, rather than ` `.
It's all up to you though.

**Comment by Rayzr522** [comment]
Don't forget to call ` ` or you might end up with memory corruption, be careful. And remember, ` ` mutates, so don't forget to clone ` `.

**Comment by Dzenly** [comment]
Look thoroughly, this leak has negative sign, so each nocode run gives you additional memory. Effect is persistent. When your memory achieves 64GB, memory will be added to your neighborhoods.

**Comment by viamgr** [comment]
It seems your memory is too low. Be aware that the project contains nothing. 

I suggest clear cache, then restart your device. Working for me...

**Comment by assuming** [comment]
Cannot reproduce this problem, please buy a new MacBook

**Comment by hanseul1795** [comment]
Dzenly//no code = peace and love, let's kill codes

**Comment by Dzenly** [comment]
@hanseul1795
The peace contains the idea of hatred, the 'no code' contains the idea of code.
I will just watch and meditate.

**Comment by hanseul1795** [comment]
@Dzenly  Wise word from the master, I'll meditate it also. ZEN!
